owncloud/ios-app
# Fixed missing check-mark images (multiple upload)

## Description
When selecting an image for upload (multiple) images, the check-mark image is missing.
Check-Mark images was deleted by merging a branch into the master.

## Related Issue
#331 

## Motivation and Context
As a user I want to see which images is selected

## How Has This Been Tested?
1. Open file list
2. Select `+` button in navigation bar
3. Select `Upload from your photo library`
4. Select an album
5. Select one or more images
6. Selected images should be marked with a blue check-mark image

## Types of changes


## Checklist:
